### DISCUSSION TITLE
Paid Consultation

### DISCUSSION THREAD

**Discussion by codemariner** [Netflix/conductor#3531] [Q&A]
My apologies if this isn't the right forum, but I don't see where there is anywhere else to ask.

We have an upcoming project supporting renewable energy that requires some workflow management. We've been evaluating Temporal but have since put our sites on Conductor. There is some pressure to hit a relatively aggressive delivery date. Would anyone with mid/large scale production experience with Conductor be able to provide some paid consultation? This could continue for a number of months as we progress.

**Comment by dilip-lukose** [comment]
Orkes (https://orkes.io/) provides an enterprise version of Conductor as a cloud service with on-prem options. This could be a good option if you are looking to run mid/large scale production workloads. Let me know if you are interested in exploring this - happy to answer any questions.
PS: also, [here is a Slack community](https://join.slack.com/t/orkes-conductor/shared_invite/zt-xyxqyseb-YZ3hwwAgHJH97bsrYRnSZg) focused on Conductor discussions that might be interesting for you to join.

**Reply by codemariner** [reply]
@dilip-lukose thank you!
